### PR TITLE
Refactor/update services names

### PR DESCRIPTION
**Pull request by BiniCodes**
- Rename all hooks that use services to `**-services.ts`
- Rename all files that use services to `**-services`
- Update all READMEs with new namings

### CODE REVIEW

**Comment by BiniCodes** [comment]
Okay I see, got to check for the types. (wait for it...)

File: examples/README.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ Here you can find some implementation examples for the [Google Maps React Hooks
...
 - [Google Map with Markers](./google-map-with-markers)
-- [Directions Service](./directions)
-- [Distance Matrix Service](./distance-matrix)
-- [Geocoding Service](./geocoding)
-- [Places Service](./places)
+- [Directions Service](./directions-service)
+- [Distance Matrix Service](./distance-matrix-service)
+- [Geocoding Service](./geocoding-service)
+- [Places Service](./places-service)
 - [Places Autocomplete Service](./places-autocomplete)
```
Comment (plumdumpling) [inline examples/README.md:11]: remove service here
```suggestion
- [Places Autocomplete](./places-autocomplete)
```

File: README.md [hunk 2]
```diff
@@ -11,10 +11,10 @@ This is a JavaScript library to easily implement a Google Maps map into your Rea
...
   - [Google Map with Markers](./examples/google-map-with-markers)
   - [Places Autocomplete](./examples/places-autocomplete)
-  - [Directions Service](./examples/directions)
-  - [Geocoding Service](./examples/geocoding)
-  - [Places Library](./examples/places)
-  - [Distance Matrix Service](./examples/distance-matrix)
+  - [Directions Service](./examples/directions-service)
+  - [Geocoding Service](./examples/geocoding-service)
+  - [Places Service](./examples/places-service)
+  - [Distance Matrix Service](./examples/distance-matrix-service)
```
Comment (plumdumpling) [inline README.md:17]: Could you please sort the examples here the same way I did in the other READMEs? It seems like I forgot this one. 😬

**Review by plumdumpling — COMMENTED** [review]
This looks all good so far. 👍 
Could you please also check the `useAutocomplete` hook? We sometimes call it "autocomplete", sometimes "places autocomplete" and maybe there's still some "places autocomplete service" (I guess it's in the library README).

File: examples/README.md (hunk 1 above)
Comment (BiniCodes) [reply examples/README.md:11]: What about `Places Autocomplete Widget` here instead? 

https://developers.google.com/maps/documentation/javascript/reference/places-widget#Autocomplete

Comment (BiniCodes) [reply examples/README.md:11]: Great 💯

File: library/docs/useAutocomplete.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # `useAutocomplete` Hook
 
-React hook to use the [Google Maps Places Autocomplete Service](https://developers.google.com/maps/documentation/javascript/reference/places-autocomplete-service) in any component.
+React hook to use the [Google Maps Places Widgets](https://developers.google.com/maps/documentation/javascript/reference/places-widget) in any component.
```
Comment (plumdumpling) [inline library/docs/useAutocomplete.md:3]:
```suggestion
React hook to use the [Google Maps Places Autocomplete Widget](https://developers.google.com/maps/documentation/javascript/reference/places-widget) in any component.
```

Comment (BiniCodes) [reply library/docs/useAutocomplete.md:3]: See #97

**Comment by BiniCodes** [comment]
PR will move to https://github.com/ubilabs/google-maps-react-hooks/pull/97

**Pull request closed without merging**